### PR TITLE
Initial codebase

### PR DESCRIPTION
**Pull request by Jedddy**
Initialized the codebase :3 🔫

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+[tool.poetry]
+name = "progphilbot"
+version = "0.1.0"
+description = ""
+authors = ["Jedddy <jedbalita25@gmail.com>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = "^3.8"
```
Comment (CunningBard) [inline pyproject.toml:9]: python 3.8 is a little old, I suggest using a higher version, and i think its better if the team only uses 1 version

File: bot/main.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        for loading databases, etc."""
+
+        print(f"{self.user.display_name} running.")
+
+    async def setup_hook(self) -> None:
+        """This method only gets called ONCE, load stuff here."""
+        # Load every cog inside cogs folder.
+        for cog in os.listdir("bot/cogs"):
+            if cog[-3:] == ".py":
+                await self.load_extension(f"cogs.{cog[:-3]}")
```
Comment (CunningBard) [inline bot/main.py:35]: I suggest adding a manual unloading, loading, and reloading commands

**Review by kylecdrck — APPROVED** [review]
lgtm

**Review by CunningBard — CHANGES_REQUESTED** [review]
The Team use only 1 Specified Python Version, We have to decide which is the most appropriate

File: pyproject.toml (hunk 1 above)
Comment (Jedddy) [reply pyproject.toml:9]: *i didnt know what everyone was using, should i change it to 3.10*

**Review by CunningBard — APPROVED** [review]
Since the changes I said arent that important, we can change it later on

File: pyproject.toml (hunk 1 above)
Comment (CunningBard) [reply pyproject.toml:9]: Yeah, 3.10 has better typing iirc

File: bot/main.py (hunk 2 above)
Comment (CunningBard) [reply bot/main.py:35]: > we can add manual unloading and loading

**Pull request merged**